src/controllers/midi/hss1394controller.h

```diff
@@ -51,7 +51,8 @@ class Hss1394Controller : public MidiController {
     virtual int close();
 
   private:
-    void sendWord(unsigned int word);
+    void sendShortMsg(unsigned char status, unsigned char byte1,
+                      unsigned char byte2);
```

Comment:
That may be the case in HSS1394Controller, but not in PortMidiController. https://github.com/mixxxdj/mixxx/pull/1004/files#diff-ab47417f917c4a5cea304eded0d68381R213
I guess it doesn't much matter since there's no engine.sendWord() function for scripts.